servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultRequestTracker.java

```diff
@@ -152,26 +150,25 @@ public final int score() {
         if (concurrentCount > 0 && concurrentStamp != Long.MIN_VALUE) {
             // If we have a request concurrent we should consider how long it has been concurrent so that sudden
             // interruptions don't have to wait for timeouts before our scores can be adjusted.
-            currentEWMA = max(currentEWMA, nanoToMillis(currentTimeNanos - concurrentStamp));
+            currentEWMA = Integer.max(currentEWMA, nanoToMillis(currentTimeNanos - concurrentStamp));
         }
 
         // Add penalty for concurrent requests to account for "unaccounted" load.
         // Penalty is the observed latency if known, else an arbitrarily high value which makes entities for which
         // no latency data has yet been received (eg: request sent but not received), un-selectable.
-        final int concurrentPenalty = (int) min(MAX_VALUE,
-                (long) concurrentCount * concurrentRequestPenalty * currentEWMA);
+        int concurrentPenalty = safeMultiply(concurrentCount, safeMultiply(currentEWMA, concurrentRequestPenalty));
```

Comment:
Side note: when I was looking at how EWMA is calculated in `updateEWMA`, it feels like if we move this part into a separate static method, it will get a higher chance for inlining:
```java
            final double tmp = (currentTimeNanos - lastTimeNanos) * invTau;
            final double w = exp(-tmp);
            nextEWMA = (int) ceil(currentEWMA * w + currentLatency * (1d - w));
```
WDYT?

Reply:
I'm not strongly opinionated one way or the other, so I included it in the latest commit.